packages/nouns-webapp/src/wagmi.ts

```diff
@@ -0,0 +1,42 @@
+import { find, pipe } from 'remeda';
+import { createConfig, http } from 'wagmi';
+import { mainnet, sepolia } from 'wagmi/chains';
+import { coinbaseWallet, injected, walletConnect } from 'wagmi/connectors';
+
+import { CHAIN_ID, WALLET_CONNECT_V2_PROJECT_ID } from './config';
+
+const activeChainId = Number(CHAIN_ID);
+
+const activeChain =
+  pipe(
+    [mainnet, sepolia],
+    find(chain => chain.id === activeChainId),
+  ) ?? sepolia;
+
+const mainnetRpcUrl = process.env.VITE_MAINNET_JSONRPC ?? '';
+const sepoliaRpcUrl = process.env.VITE_SEPOLIA_JSONRPC ?? '';
```

Comment:
```suggestion
const mainnetRpcUrl = import.meta.env.VITE_MAINNET_JSONRPC ?? '';
const sepoliaRpcUrl = import.meta.env.VITE_SEPOLIA_JSONRPC ?? '';
```

those are currently not being populated